src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__content">
+        <img 
+          src="./images/imac.jpeg" 
+          alt="imac image" 
+          class="card__product-image"
+        >
+
+        <p class="card__product-name">
+          APPLE A1419 iMac 27&#34; Retina 5K Monoblock &#40;MNED2UA&#47;A&#41;
+        </p>
+
+        <p class="card__product-code">
+          Product code&#58; 195434
+        </p>
+
+        <div class="card__stars">
+          <div class="stars">
+            <ul class="stars__item stars__item--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <span class="stars__reviews">Reviews&#58; 5</span>
+          </div>
+        </div>
```

Comment:
```suggestion
        <div class="card__reviews">
          <div class="card__stars">
            <ul class="stars stars--4">
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
            </ul>

            <span class="card__reviews-text">Reviews&#58; 5</span>
          </div>
        </div>
```
let`s make it work this way)